### PR TITLE
Landing Page for Sysdig Charts - DOC-2557

### PR DESCRIPTION
**Pull request by radhikapc**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x ] Title of the PR starts with chart name (e.g. [mychartname])
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ]  Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Review by achandras — APPROVED** [review]
lgtm!

File: index.md [hunk 1]
```diff
@@ -3,37 +3,43 @@ title: Sysdig Helm Charts
...
+|[`cloud-connector`](/charts/cloud-connector) | Deploys the [Sysdig Cloud Connector](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/) in your Kubernetes cluster to enable threat-detection and image scanning. |
+|[`harbor-scanner-sysdig-secure`](/charts/harbor-scanner-sysdig-secure) | Deploys the Harbor Scanner Adapter to enable Harbor to use Sysdig Secure scanning engine to analyze the container images managed by the platform. |
+|[`registry-scanner`](/charts/registry-scanner) | Deploys the Sysdig Registry Scanner as a scheduled cronjob in your Kubernetes cluster. |
+|[`sysdig-mcm-navmenu`](/charts/sysdig-mcm-navmenu) | Deploys services and ingress required to create Multi-Cloud Management menu in Sysdig. |
+|[`sysdig-stackdriver-bridge`](/charts/sysdig-stackdriver-bridge) | Deploys the the Sysdig Stackdriver Bridge application to enable reading and sending audit logs from Stackdriver to Sysdig agent. |
+## Add Chart Repository
```
Comment (achandras) [inline index.md]: This might need a newline above, I don't remember exactly but I think without it there are rendering issues

File: index.md [hunk 2]
```diff
@@ -3,37 +3,45 @@ title: Sysdig Helm Charts
...
-## Add this charts repository to your local Helm
+Helm is an open-source package manager for Kubernetes applications. Using a helm chart you can describe, install, configure, and upgrade applications automatically.
 
-To start using our charts, first add this _charts repository_ using the `helm` command line tool:
+To simplify your deployment experience, Sysdig provides helm charts to install and configure Sysdig installation components in your Kubernetes environment.
+
+
+## Sysdig Chart Repository
+
+The Sysdig Chart Repository includes the following charts:
```
Comment (wideawakening) [inline index.md:13]: due to removing grammatical list of charts, i would maintain deprecated charts too but in the lower part of the page, and marked as deprecated.

File: index.md [hunk 3]
```diff
@@ -3,37 +3,45 @@ title: Sysdig Helm Charts
...
+|[`sysdig-deploy`](/charts/sysdig-deploy)    | Deploys the following components in your Kubernetes cluster:<br>`kspm-collector`:  Enables collecting and sendin Kubernetes resource manifests to be evaluated against compliance policies. The scan results are displayed by Sysdig Secure's Actionable Compliance.<br>`node-analyzer`: Enables  Image Analyzer, Benchmarks, and Host Scanning features.<br>`rapid-response`:  Allows designated advance users to investigate and troubleshoot events from a remote shell.<br>`agent`: Processes syscall events and metrics, creates [capture](https://docs.sysdig.com/en/docs/sysdig-secure/investigate/captures/#captures) files, and performs auditing and compliance. |
+|[`admission-controller`](/charts/admission-controller) | Deploys the [Sysdig Admission Controller](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/) in your Kubernetes cluster to enable image scanning and audit logging. |
+|[`cloud-connector`](/charts/cloud-connector) | Deploys the [Sysdig Cloud Connector](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/) in your Kubernetes cluster to enable threat-detection and image scanning. |
```
Comment (wideawakening) [inline index.md:19]: for the descriptions link of Cloud-Connector i would specifically point to this link
https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/#helm-chart-based-feature-limited

Comment (wideawakening) [inline index.md:19]: for the descriptions link of Cloud-Connector i would specifically point to this link
https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/#helm-chart-based-feature-limited

**Comment by aroberts87** [comment]
~@wideawakening - are you formally requesting changes, or can this be merged?~
Apologies. I'm going to merge this and we can address any concerns in a follow up PR.

**Comment by wideawakening** [comment]
> ~@wideawakening - are you formally requesting changes, or can this be merged?~ Apologies. I'm going to merge this and we can address any concerns in a follow up PR.

i was just suggesting :)
no blockers.

i'm ok with leaving deprecated charts out of the table.
still think for cloud-connector link my suggestion docs url makes more sense

**Pull request merged**